GUIDELINES.md

```diff
@@ -0,0 +1,135 @@
+# sweetest test development guidelines
+
+After some time working with sweetest we came up to the conclusion that it leaves a lot of freedom to design tests as we want. But on the other side there is an increasing need for alignment. These guidelines are here for reaching an appropriate level of alignment. Feel free to challenge the current state and to contribute!
+
+## Content
+
+* [Goals](#goals)
+* [One-by-one steps](#one-by-one-steps)
+* [Principles](#principles)
+* [Links](#links)
+
+## Goals
+
+* Put a **layer of abstraction** on the system under test: we call these abstractions _steps_ (as derived from Cucumber)
+  * so if the system under test changes, the test system doesn't need to change
+  * so the test just tells _what_ is tested, not _how_ (tests become more business-centric, whenever possible, all the details are in the steps)
+  * so you can reuse test code
```

Comment:
Which one exactly? The _business-centric_ or the _technical implementation_ ones?

Reply:
the only thing you can reuse is the steps class, so the technical implementation. should i outline that clearer?